subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/daemon/DaemonLogsAnalyzer.groovy

```diff
@@ -56,6 +56,7 @@ class DaemonLogsAnalyzer implements DaemonsFixture {
     }
 
     void killAll() {
+        printCrashLogIfAny()
```

Comment:
I would say we should fail if we find a crash log, not only print it. All the tests which expect the daemon to be crashing should declare that.

Reply:
It will effectively fail on TeamCity, but let's make it explicit.

Reply:
Looks like even with the assertion, the test won't fail since all exceptions from cleaning up isolated daemons are swallowed: https://github.com/gradle/gradle/blob/70a69609fb801ba3ffd8e7e3cdf645ce7e514019/subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/AbstractGradleExecuter.java#L848-L856

Maybe the detection of the crash logs needs to be separate?